types/vscode/index.d.ts

```diff
@@ -1735,12 +1735,20 @@ declare module 'vscode' {
          * like "TypeScript", and an array of extensions, e.g.
          * ```ts
          * {
-         *     'Images': ['png', 'jpg']
-         *     'TypeScript': ['ts', 'tsx']
+         * 	'Images': ['png', 'jpg']
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/vscode/index.d.ts;
  Line: 1738;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.